app/assets/stylesheets/pages/_home.scss

```diff
@@ -1 +1,64 @@
 // Specific CSS for your home-page
+.banner {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 100vh;
+  top: 0;
+
+}
+
+.titles {
+  color: $gold;
+
+  h1 {
+    font-size: 150px;
+    font-weight: bold;
+    font-family: $title-font;
+    text-shadow: 5px 5px 5px rgb(14, 13, 13);
+  }
+
+  h3 {
+    font-size: 30px;
+    color: white;
+    font-family: $body-font;
+  }
+
+  .btn-index {
+    display: flex;
+    justify-content: flex-end;
+    margin-top: 50px;
+  }
+
+}
+
+.image {
+  opacity: 0.8;
+  box-shadow: 5px 5px 10px rgb(14, 13, 13);
+  width: 375px;
+
+}
```

Comment:
.image should be more easy to understand. Can you rename it to something like "banner-painting" or something like that ?

Also there is an empty line in l:39